tests/foreman/cli/test_myaccount.py

```diff
@@ -108,23 +116,15 @@ def test_positive_update_language(self):
 
         @caseautomation: notautomated
         """
-
-    @tier1
-    def test_positive_update_password(self):
-        """Update Password in My Account
-
-        @id: e7e9b212-f0aa-4f7e-8433-b4639da89495
-
-        @Assert: User is updated
-        """
-        password = gen_string('alphanumeric')
-        User.update({
-            'id': self.user['id'],
-            'password': password,
-        })
-        User.foreman_admin_password = password
-        result = User.info({'id': self.user['id']})
-        self.assertTrue(result)
+        locales = [
```

Comment:
please use constant instead. If no one of them suffice your needs, please create your own